### PR TITLE
Create build-and-release-kanvas.yml

### PR DESCRIPTION
**Pull request by MUzairS15**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/build-and-release-kanvas.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          repository: layer5labs/meshery-extensions
+          path: "meshery-extensions"
+          fetch-depth: 1
+          ref: ${{ inputs.branch }}
+      
+      - name: Checkout meshery-cloud 
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN_CLOUD }}
```
Comment (MUzairS15) [inline .github/workflows/build-and-release-kanvas.yml:41]: I added this token randomly, this doesn't exist currently.

We might need a GH pat for the cloud repo added as a secret.

**Comment by MUzairS15** [comment]
After this gets merged the "Notify Remote Providers" workflow in meshery/meshery needs to be updated.

File: .github/workflows/build-and-release-kanvas.yml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+      - name: Setup Node 16
+        uses: actions/setup-node@v4
+        with:
+          node-version: '16'
+
+      - name: Build extension
+        run: |
+          export PROVIDER_PATH="provider/navigator";
+          make prod;
```
Comment (leecalcote) [inline .github/workflows/build-and-release-kanvas.yml:79]:
```suggestion
          make kanvas-prod;
```

Just `prod` has been previously deprecated.

File: .github/workflows/build-and-release-kanvas.yml (hunk 1 above)
Comment (leecalcote) [inline .github/workflows/build-and-release-kanvas.yml:41]:
```suggestion
          token: ${{ secrets.GH_ACCESS_TOKEN }}
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**